frontend/src/v4/routes/components/colorPicker/colorPicker.component.tsx

```diff
@@ -398,13 +399,14 @@ export class ColorPicker extends PureComponent<IProps, IState> {
 		<Footer>
 			<StyledButton
 				variant="contained"
-				color="secondary"
+				color={isV5() ? 'primary' : 'secondary'}
 				onClick={this.handleSave}
 			>
 				Save
 			</StyledButton>
 			<StyledButton
-				color="primary"
+				color={isV5() ? 'secondary' : 'primary'}
+				variant="outlined"
```

Comment:
This changes the button in v4 too. Has this been requested?

Reply:
fixed